connectivity/nanostack/mbed-mesh-api/source/NanostackPPPInterface.cpp

```diff
@@ -334,7 +334,7 @@ void PPPPhy::link_state_cb(bool up)
     }
 }
 
-void PPPPhy::get_iid64(uint8_t *iid64)
+void PPPPhy::get_iid64(uint8_t *iid64_)
```

Comment:
You changed the argument to `iid64_` but it's not used in the function.

Reply:
thank you thats a huge mistake

Reply:
That being said, I don't like the trailing `_`, I think it's fine to keep the `iid64`

Reply:
yes i am with you but i had to come up with a not so visible way to cope with the wshaddow warnings
i think the trailing _ complies with the mbed codestyle (at least i could not find any documentaion stating something different) 
since more than 90% of the warnings were wshaddow and i fixed nearly all of them with a trailing _ in the arguments the PR would become obsolete
but i agree that thats not consitent with the other argument names.
if we do not want that i would close the PR and we should work out a more detailed documentation on variable namings